contracts/cheddar_5x5_tic_tac_toe/src/lib.rs

```diff
@@ -233,12 +246,42 @@ impl Contract {
         }
     }
 
-    pub fn get_last_move(&self, game_id: &GameId) -> (Option<Coords>, Piece){
+    pub fn get_last_move(&self, game_id: &GameId) -> (Option<Coords>, Piece, Option<GameResult>, Option<u32>){
```

Comment:
please document what is the returned value means (u32).

Reply:
Done